node/pkg/processor/cleanup.go

```diff
@@ -228,7 +228,10 @@ func (p *Processor) handleCleanup(ctx context.Context) {
 					if err := common.PostObservationRequest(p.obsvReqSendC, req); err != nil {
 						p.logger.Warn("failed to broadcast re-observation request", zap.String("message_id", s.LoggingID()), zap.Error(err))
 					}
-					p.gossipSendC <- s.ourMsg
+					p.postObservationToBatch(s.ourObs)
+					if s.ourMsg != nil {
+						p.gossipSendC <- s.ourMsg // TODO: Get rid of this
```

Comment:
Are you getting rid of this in a future PR?

Reply:
Yes, after the cutover is complete.